src/mappings/mappingHandlers.ts

```diff
@@ -47,6 +51,10 @@ export async function handleBlock(block: SubstrateBlock): Promise<void> {
     if (!dbBlock) {
       // Generic block
       if (ENABLE_LOG) logger.info(`Block handler: N°${blockNumberString}`)
+      // Override block timestamp
+      if (!block.timestamp){
+        block.timestamp = new Date()
```

Comment:
Added a comment to explain, it was to satisfy the compiler since in `SubstrateBlock` timestamp was now optional (of course not for us)

I removed this condition and the explanation is on the interface : 
```ts
// Set timestamp as mandatory, since in `SubstrateBlock` timestamp is now an optional field.
// For us, it will never be empty and this is only to satisfy the compiler.
export interface CorrectSubstrateBlock extends SubstrateBlock {
  timestamp: Date;
}
```